src/torchcodec/decoders/_audio_decoder.py

```diff
@@ -13,7 +13,7 @@
 from torchcodec.decoders import _core as core
 from torchcodec.decoders._decoder_utils import (
     create_decoder,
-    get_and_validate_stream_metadata,
+    ERROR_REPORTING_INSTRUCTIONS,
```

Comment:
Changes below: we previously had a common util for audio and video that extracted and validated the metadata: `get_and_validate_stream_metadata()`. Since we moved a bunch of fields as video-only, this util wasn't generic enough anymore to justify its existence, hence a few edits here and in the `video_decoder.py` file.